resources/[qb]/[shops]/indigo-masks/shopconfig.txt
paste this in to indigo-shops/config.lua 

    ["masker"] = {
        [1] = {
            name = "terror",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 1,
        },
        [2] = {
            name = "hockeymask",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 2,
        },
        [3] = {
            name = "blackbandana",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 3,
        },
        [4] = {
            name = "tshirtmask",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 4,
        },
        [5] = {
            name = "scarecrowmask",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 5,
        },
        [6] = {
            name = "scarf",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 6,
        },
        [7] = {
            name = "monkeymask",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 7,
        },
        [8] = {
            name = "skullmask",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 8,
        },
        [9] = {
            name = "terrorwit",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 9,
        },
        [10] = {
            name = "feest",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 10,
        },
        [11] = {
            name = "noselong",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 11,
        },
        [12] = {
            name = "blauw",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 12,
        },
        [13] = {
            name = "skullzwart",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 13,
        },
        [14] = {
            name = "silver",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 14,
        },
        [15] = {
            name = "zwart1",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 15,
        },
        [16] = {
            name = "zwart2",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 16,
        },
        [17] = {
            name = "please",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 17,
        },
        [18] = {
            name = "sok",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 18,
        },
        [19] = {
            name = "zak",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 19,
        },
        [20] = {
            name = "groen",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 20,
        },
        [21] = {
            name = "bandanab",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 21,
        },
        [22] = {
            name = "bivak1",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 22,
        },
        [23] = {
            name = "roodc",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 23,
        },
        [24] = {
            name = "clown",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 24,
        },
        [25] = {
            name = "gorilla",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 25,
        },
        [26] = {
            name = "paard",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 26,
        },
        [27] = {
            name = "paardu",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 27,
        },
        [28] = {
            name = "skullr",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 28,
        },
        [29] = {
            name = "masker3",
            price = 500,
            amount = 10,
            info = {},
            type = "item",
            slot = 29,
        },
        [30] = {
            name = "helm1",
            price = 1000,
            amount = 10,
            info = {},
            type = "item",
            slot = 30,
        },
        [31] = {
            name = "helm2",
            price = 1000,
            amount = 10,
            info = {},
            type = "item",
            slot = 31,
        },
        [32] = {
            name = "helm3",
            price = 1000,
            amount = 10,
            info = {},
            type = "item",
            slot = 32,
        },
        [33] = {
            name = "helm4",
            price = 1000,
            amount = 10,
            info = {},
            type = "item",
            slot = 33,
        },
        [34] = {
            name = "helm5",
            price = 1000,
            amount = 10,
            info = {},
            type = "item",
            slot = 34,
        },
        [35] = {
            name = "helm6",
            price = 1000,
            amount = 10,
            info = {},
            type = "item",
            slot = 35,
        },
        [36] = {
            name = "helm7",
            price = 1000,
            amount = 10,
            info = {},
            type = "item",
            slot = 36,
        },
        [37] = {
            name = "helm8",
            price = 1000,
            amount = 10,
            info = {},
            type = "item",
            slot = 37,
        },
    },

en this all the way below in indigo-shops/config.lua

-- OLD STORE 

    ["masker"] = {
        ["label"] = "Mask shop",
        ["coords"] = {
            [1] = vector4(-1334.89, -1277.6, 4.96, 105.15)
        },
        ["ped"] = {
            ["model"] = 'S_M_M_StrVend_01'
        },
        ["radius"] = 4.5,
        ["products"] = Config.Products["masker"],
        ["showblip"] = true,
        ["blipsprite"] = 84
    },

-- VB MARKET MASK SHOP GABZZ 

    ["masker"] = {
        ["label"] = "Mask shop",
        ["coords"] = {
            [1] = vector4(-1198.76, -1515.58, 4.38, 124.93)
        },
        ["ped"] = {
            ["model"] = 'u_m_y_pogo_01'
        },
        ["radius"] = 4.5,
        ["products"] = Config.Products["masker"],
        ["showblip"] = false,
        ["blipsprite"] = 84
    },



resources/[qb]/[ui]/dz-drone/config.lua
Config = {}

Config.Framework = "qbcore" -- Set you framework "qbcore" or "esx" or "" if you use standalone framework and use /drone 1 or 2 command to use the Drone
Config.QBCoreName = "indigo-core" -- You QBCore script name must be correct to work (only if you use QBCore Framework)
Config.IsESXLegacy = false -- Set it true is you're using esx legacy (only if you use ESX Framework)

Config.Controls = { -- FiveM Controls: https://docs.fivem.net/docs/game-references/controls/
	Forward		= 32,	-- W for Qwerty / Z for Azerty
	Backward	= 33,	-- S
	Left		= 34,	-- A for Qwerty / Q for Azerty
	Right		= 35,	-- D
	Up			= 51,	-- E
	Down		= 52,	-- Q for Qwerty / A for Azerty
	Stop		= 22,	-- Space
	ZoomOut		= 16,	-- Mouse Scroll Whell Down
	ZoomIn		= 17,	-- Mouse Scroll Whell Up
	Nightvision	= 140,	-- R
	Heatvision	= 75,	-- F
	Spotlight	= 47,	-- G
	Scanner		= 24,	-- Left Mouse Button
	Cancel		= 200,	-- ESC
}

Config.Transition = {
	['direction']       	= 'Direction',
	['height']          	= 'Height',
	['camera']          	= 'Camera',
	['zoom']            	= 'Zoom',
	['nightvision']     	= 'Nightvision',
	['heatvision']      	= 'Heatvision',
	['spotlight']       	= 'Spotlight',
	['scan_player']     	= 'Scan Player',
	['cancel']          	= 'Cancel',
	['cant_use_drone']  	= 'You can\'t use the drone',
	
	['scan_searching']  	= 'Searching...',
	['scan_searching_db']	= 'Searching in database...',
	['scan_unknown']		= 'Unknown',
	['scan_not_recognized']	= 'Target cannot be recognized',
}